src/parseTree/nodes/str_literal.java

```diff
@@ -46,15 +41,16 @@ public void fixChildren() {
     public str_literal() {
     }
 
-    public str_literal(quote preQuote, str_token data, quote endQuote) {
-        if (preQuote == null || data == null || endQuote == null) {
+    public void str_literal_set(quote preQuote, str_token data, quote endQuote, id alias) {
+        if (preQuote == null && data == null && endQuote == null && alias == null) {
```

Comment:
Without quotes, this conditional may be `if (data == null && alias == null) {` which IMO makes more sense. I see how it works though

Reply:
That was just a bodge to get it to fix children